### PR TITLE
fix(terraform): optimize check CKV2_AWS_60

### PR DESCRIPTION
**Pull request by praveen-panw**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

This PR contains modified YAML file for **CKV2_AWS_60: Ensure RDS instance with copy tags to snapshots is enabled**

### 1. Checkov Title - Ensure RDS instance with copy tags to snapshots is enabled

<ins>PC Policy ID</ins> - 1f524c07-3254-45a0-8ad7-03e29242c499

<ins>PC Policy Title</ins> - AWS RDS instance with copy tags to snapshots disabled

<ins>Compliance standards </ins> - 

`CIS Controls v7.1` `AWS Foundational Security Best Practices standard` `CSA CCM v.4.0.1` `MAS TRM 2021` `Copy of APRA (CPS 234) Information Security` `ISO/IEC 27002:2013` `mkurter-CIS Controls v8-testing` `MLPS 2.0 (Level 2)` `CIS Controls v8` `HITRUST CSF v.9.6.0` `ACSC Information Security Manual (ISM)` `Secure Controls Framework (SCF) - 2022.2.1` `Copy of ACSC Information Security Manual (ISM)` `Sarbanes Oxley Act (SOX)` `APRA (CPS 234) Information Security` `Cybersecurity Maturity Model Certification (CMMC) v.1.02` `Brazilian Data Protection Law (LGPD)` `NIST SP 800-171 Revision 2` `PCI DSS v3.2.1` `NIST SP 800-172` `HITRUST v.9.4.2` `NIST CSF` `ISO/IEC 27017:2015` `ISO/IEC 27018:2019`

<ins>Remediation:</ins>

1. Sign into the AWS console
2. In the console, select the specific region from region drop down on the top right corner, for which the alert is generated
3. Navigate to Amazon RDS console
4. Choose DB Instances, and then select the reported DB instance
5. Click on 'Modify'
6. In 'Additional Configuration' section, In 'Backup' sub-section select the 'Copy tags to snapshots'
7. Click on 'Continue'
8. On the 'Summary of Modifications' panel, review the configuration changes. From 'Scheduling of Modifications' section, select whether changes to 'Apply immediately' or 'Apply during the next scheduled maintenance window'.
9. On the confirmation page, Review the changes and Click on 'Modify DB Instance' to save your changes.

---

### CODE REVIEW

**Comment by praveen-panw** [comment]
Hello @gruebel,

I've modified the YAML file like we discussed. Feel free let me know if there's any changes required. 🙂

File: checkov/terraform/checks/graph_checks/azure/AzureConfigMSSQLwithAD.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+metadata:
+  id: CKV2_AZURE_27
```
Comment (gruebel) [inline checkov/terraform/checks/graph_checks/azure/AzureConfigMSSQLwithAD.yaml:2]: I think you accidentally committed new checks to this PR 😄

Comment (praveen-panw) [reply checkov/terraform/checks/graph_checks/azure/AzureConfigMSSQLwithAD.yaml:2]: Uh oh. Sorry, will be updating it now.

Comment (praveen-panw) [reply checkov/terraform/checks/graph_checks/azure/AzureConfigMSSQLwithAD.yaml]: Done. Kindly check now

**Review by gruebel — APPROVED** [review]
thanks 💯

**Review by ChanochShayner — APPROVED** [review]

**Pull request merged**